modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/store/FilePageStore.java

```diff
@@ -156,10 +138,18 @@ public void stop(boolean clean) throws IgniteInternalCheckedException {
 
     /** {@inheritDoc} */
     @Override
-    public long allocatePage() throws IgniteInternalCheckedException {
+    public int allocatePage() throws IgniteInternalCheckedException {
         ensure();
 
-        return pageCount.getAndIncrement();
+        int pageIdx = pageCount.getAndIncrement();
+
+        PageAllocationListener listener = this.pageAllocationListener;
+
+        if (listener != null) {
+            listener.onAllocationPage(pageIdx);
```

Comment:
"onAllocationPage" is grammatically incorrect. Please use better names, like "onPageAllocated", for example. Don't ignore proper English in naming

Reply:
Fix it.